### DISCUSSION TITLE
Weather System

### DISCUSSION THREAD

**Discussion by WolfieFukustuno** [sunflower-land/sunflower-land#1133] [Ideas]
If stamina appears in the form of water in the expansion of the lands, and if weather is added to the game, then I have a little idea. What if during the rain there is a boost for all players, which increases the growth rate of plants by any %. In addition to all this, a bucket of water can also be filled with some small part of it. This will make it easier (if this relief is of course needed) for players to earn more sfl. Let the rain be a phenomenon that occurs spontaneously, for example, 3 times a week for 4 hours. This will not really ruin the economy, but not everyone will be able to take advantage of it, because for example, you plant carrots and wait for an hour, then you forget about the game for a hour, but because of the rain, carrots will grow for example one and a half times faster. Only after an hour you will come on farm and realize that rain is falling and in fact you could have come in earlier, harvested and planted something that grows longer. You can add a different % boost to each of the plant species separately. For example, you can increase wheat growth by 2 timesby 4 hours, but for example carrots are grows only 50% faster, and potatoes are even 15% (I want people to have motivation to plant crops that grow the longer than others. The longer the vegetable grows, the stronger the boost will be during the rain).

I also suggest adding a voodoo-goblin to the goblin village (if it goblin village be in the expansion of the lands of course) which can summon rain and every farmer could interact with him. In order for him to cause rain, you need to give him a certain amount of sfl, like 1000 maybe and interesting part about it, that every farmer could give coins to him (by burning it or giving 1000 slf to the team). You can give whatever amount of coins you want. When the scale is filled, the voodoo-goblin will dance and sing, after that rain will fall on all the farmers lands.

**Comment by InuBakaBo** [comment]
Additionally if water is like a stamina system for crops etc, then we can also have water tanks to collect rainwater. Kinda like Solar Panels in Million on Mars
